diff -rup lohit-devanagari-2.4.3/66-lohit-devanagari.conf lohit-devanagari-2.4.3_mod/66-lohit-devanagari.conf
--- lohit-devanagari-2.4.3/66-lohit-devanagari.conf	2010-02-04 13:49:05.000000000 +0530
+++ lohit-devanagari-2.4.3_mod/66-lohit-devanagari.conf	2010-04-16 10:34:43.000000000 +0530
@@ -8,7 +8,7 @@
   <test name="family">
     <string>sans-serif</string>
   </test>
-  <edit name="family" mode="prepend" binding="same">
+  <edit name="family" mode="prepend">
     <string>Lohit Devanagari</string>
   </edit>
 </match>
@@ -20,7 +20,7 @@
   <test name="family">
     <string>sans-serif</string>
   </test>
-  <edit name="family" mode="prepend" binding="same">
+  <edit name="family" mode="prepend">
     <string>Lohit Devanagari</string>
   </edit>
 </match>
@@ -32,7 +32,7 @@
   <test name="family">
     <string>sans-serif</string>
   </test>
-  <edit name="family" mode="prepend" binding="same">
+  <edit name="family" mode="prepend">
     <string>Lohit Devanagari</string>
   </edit>
 </match>
@@ -44,7 +44,7 @@
   <test name="family">
     <string>sans-serif</string>
   </test>
-  <edit name="family" mode="prepend" binding="same">
+  <edit name="family" mode="prepend">
     <string>Lohit Devanagari</string>
   </edit>
 </match>
@@ -56,7 +56,7 @@
   <test name="family">
     <string>sans-serif</string>
   </test>
-  <edit name="family" mode="prepend" binding="same">
+  <edit name="family" mode="prepend">
     <string>Lohit Devanagari</string>
   </edit>
 </match>
@@ -68,7 +68,7 @@
   <test name="family">
     <string>sans-serif</string>
   </test>
-  <edit name="family" mode="prepend" binding="same">
+  <edit name="family" mode="prepend">
     <string>Lohit Devanagari</string>
   </edit>
 </match>
@@ -80,7 +80,7 @@
   <test name="family">
     <string>sans-serif</string>
   </test>
-  <edit name="family" mode="prepend" binding="same">
+  <edit name="family" mode="prepend">
     <string>Lohit Devanagari</string>
   </edit>
 </match>
@@ -92,7 +92,7 @@
   <test name="family">
     <string>sans-serif</string>
   </test>
-  <edit name="family" mode="prepend" binding="same">
+  <edit name="family" mode="prepend">
     <string>Lohit Devanagari</string>
   </edit>
 </match>
@@ -104,7 +104,7 @@
   <test name="family">
     <string>sans-serif</string>
   </test>
-  <edit name="family" mode="prepend" binding="same">
+  <edit name="family" mode="prepend">
     <string>Lohit Devanagari</string>
   </edit>
 </match>
@@ -116,7 +116,7 @@
   <test name="family">
     <string>sans-serif</string>
   </test>
-  <edit name="family" mode="prepend" binding="same">
+  <edit name="family" mode="prepend">
     <string>Lohit Devanagari</string>
   </edit>
 </match>
